packages/langium/package.json

```diff
@@ -24,7 +24,7 @@
     "watch": "tsc --watch",
     "lint": "eslint ./src --ext .js,.ts",
     "test": "jest --coverage",
-    "langium:generate": "node ../langium-cli/bin/langium generate",
+    "langium:generate": "langium generate",
```

Comment:
Shouldn't we keep the local path to ensure that we use the local version?

Reply:
We don't need it, since npm workspaces has no issues with cyclic dependencies. So the `langium` call will always go to the local `langium-cli` package.

Reply:
Ok, but please change the dependency to a fixed version: `"langium-cli": "0.1.0"` (without the `^`)